R/module_filter_manager.R

```diff
@@ -1,9 +1,33 @@
+#' Manage multiple `FilteredData` objects
+#'
+#' Oversee filter states in the whole application.
+#'
+#' This module observes the changes of the filters in each `FilteredData` object
```

Comment:
Q: Is this supposed to be in the details section?

Just resolve the comment if it is 😁

![image](https://github.com/insightsengineering/teal/assets/211358/d096b724-8e90-4046-adc3-a183899ce0d0)

Reply:
yes